elide-core/src/main/java/com/yahoo/elide/Elide.java

```diff
@@ -190,9 +198,14 @@ public ElideResponse post(String path, String jsonApiDocument, User opaqueUser,
             JsonApiDocument jsonApiDoc = mapper.readJsonApiDocument(jsonApiDocument);
             RequestScope requestScope = new RequestScope(path, apiVersion,
                     jsonApiDoc, tx, user, null, elideSettings);
-            requestScope.setEntityProjection(new EntityProjectionMaker(elideSettings.getDictionary(),
-                    requestScope).parsePath(path));
+            EntityProjection projection = new EntityProjectionMaker(elideSettings.getDictionary(),
+                    requestScope).parsePath(path);
+            requestScope.setEntityProjection(projection);
             BaseVisitor visitor = new PostVisitor(requestScope);
+
+            //accept and process the query here
```

Comment:
This comment string does not add much value as the two functions names describe exactly this comment.